dashboard/package.json

```diff
@@ -37,6 +37,13 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
+  "jest":{
+    "transform": {
+      "^.+\\.[t|j]sx?$": "babel-jest"
```

Comment:
No; the first one is an escape for the second one; the resulting single backslash is then an escape for the subsequent dot, so that it is a literal period instead of a wildcard character.